composable_kernel/include/utility/config.hpp

```diff
@@ -59,14 +59,18 @@
 #define CK_USE_AMD_INNER_PRODUCT_INLINE_ASM 1
 #endif
 
-// AMD buffer addressing
-#ifndef CK_USE_AMD_BUFFER_ADDRESSING
-#define CK_USE_AMD_BUFFER_ADDRESSING 1
+// AMD buffer_load. TODO: conv3d does NOT work with buffer load
+#ifndef CK_USE_AMD_BUFFER_LOAD
+#define CK_USE_AMD_BUFFER_LOAD 0
```

Comment:
We cannot turn off buffer_load for all other kernels because of conv3d

Please turn this on by default, and open a compiler ticket 

We will turn off conv3d by default until buffer_load is fixed or until we have a workaround

Reply:
Make sense to me.